compiler/lcalc/to_ocaml.ml

```diff
@@ -17,6 +17,19 @@ open Ast
 open Backends
 module D = Dcalc.Ast
 
+
+let find_struct s ctx =
+  try
+    D.StructMap.find s ctx.D.ctx_structs
+  with Not_found -> Errors.raise_spanned_error (Format.sprintf "Internal Error: Structure blah was not found in the current environment.") Pos.no_pos
```

Comment:
Replace `Pos.no_pos` with the information in `s` as you did for `find_enum`?

Reply:
fixed